BOJ/1647_김덕래.cpp

```diff
@@ -0,0 +1,71 @@
+#include<iostream>
+#include<algorithm>
+#include<cmath>
+#include<vector>
+#include <queue>
+using namespace std;
+
+struct Edge{
+    int n1, n2;
+    double dist;
+    Edge(int n1_, int n2_, double dist_) : n1(n1_), n2(n2_), dist(dist_) { }
+    friend bool operator<(const Edge& e1, const Edge& e2){
+        return e1.dist < e2.dist;
+    }
```

Comment:
```suggestion
    friend bool operator<(const Edge& e1, const Edge& e2){
        return e1.dist < e2.dist;
    }
```
여기서 friend는 어떤 역할인가요?

Reply:
그냥 인터넷에서 오버로딩 코드 긁어왔어요... ㅠㅠ